src/App.css

```diff
@@ -612,13 +612,13 @@ h1 .num-contexts {
 
 .child > .bullet {
   transition: transform 0.1s ease-in-out;
+  margin-left: -4.4px;
```

Comment:
Please put all styles on the `.bullet::before` element, otherwise the animation will be affected.